packages/components-native/src/InputText/context/InputAccessory.style.ts

```diff
@@ -7,7 +7,7 @@ export const styles = StyleSheet.create({
     justifyContent: "space-between",
     alignItems: "center",
     paddingHorizontal: tokens["space-small"],
-    backgroundColor: tokens["color-surface--background"],
+    backgroundColor: tokens["color-interactive--background"],
```

Comment:
```suggestion
    backgroundColor: tokens["color-surface--background"],
```

Fairly sure this is related to the same element as the previous file